be/src/olap/rowset/segment_v2/column_writer.h

```diff
@@ -38,6 +39,7 @@ struct ColumnWriterOptions {
     CompressionTypePB compression_type = NO_COMPRESSION;
     bool need_checksum = false;
```

Comment:
It's not related to this PR, but I think `need_checksum` should be removed. Reader can choose to ignore or verify checksum but writer should always compute and store checksum for each page.

Reply:
Yeah, in next PR, I will support checksum in page, then I will rethink how we support checksum.

Reply:
OK

Reply:
already changed. All write will compute checksum and save it